.github/composite-actions/run-pg-upgrade/action.yml

```diff
@@ -32,11 +32,13 @@ runs:
         echo 'Updating babelfish extensions...'
         cd ~/work/babelfish_extensions/babelfish_extensions/
         ~/${{ inputs.pg_new_dir }}/bin/pg_ctl -D ~/${{ inputs.pg_new_dir }}/data -l ~/${{ inputs.pg_new_dir }}/data/logfile14 start
+        echo 'Set the migration_mode before running the upgrade scripts..'
+        sudo ~/${{ inputs.pg_new_dir }}/bin/psql -d jdbc_testdb -U runner -c "ALTER SYSTEM SET babelfishpg_tsql.migration_mode = '${{inputs.migration_mode}}';"
+        sudo ~/${{ inputs.pg_new_dir }}/bin/psql -d jdbc_testdb -U runner -c "SELECT pg_reload_conf();"
```

Comment:
In case of major version upgrade, after pg_upgrade the migration mode was set to default which is hard-coded as single-db and later we update the extensions and set the migration mode as it was before the MVU. 
The upgrade script change in this PR expects the original migration mode at the time of executing "update_catalog_for_guest" function, which becomes untrue with the previous implementation. 
That's the reason I have set the migration mode before running the upgrade scripts.